src/Codelens/CodelensProvider.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as vscode from 'vscode';
+
+export class CodelensProvider implements vscode.CodeLensProvider {
+  constructor() {}
+
+  public provideCodeLenses(document: vscode.TextDocument, token: vscode.CancellationToken):
+      vscode.CodeLens[]|Thenable<vscode.CodeLens[]> {
+    return [];
```

Comment:
It would be better to leave a TODO comment. And, posting a full codes draft would be helpful as well.
```suggestion
	// TODO implement
    return [];
```

Reply:
Should we annotate what we have to do? Or should I just leave the "TODO implement" as it is?